sqeduler.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "pry", "~> 0"
   gem.add_development_dependency "rake", "~> 10"
   gem.add_development_dependency "rspec", "~> 3.3"
-  gem.add_development_dependency "rubocop", "~> 0.24"
```

Comment:
This definition allows for any gem >1.0 whereas if you said `~> 0.24.0`, it would have allowed only for the gems >0.25.
http://stackoverflow.com/a/5170592/4951255